lib/targets.go

```diff
@@ -171,12 +169,18 @@ func NewLazyTargeter(src io.Reader, body []byte, hdr http.Header) Targeter {
 	}
 }
 
-var httpMethodChecker = regexp.MustCompile("^(HEAD|GET|PUT|POST|PATCH|OPTIONS|DELETE) ")
+var httpMethodChecker = regexp.MustCompile("^[A-Z]*$")
 
-func startsWithHTTPMethod(t string) bool {
+func isHttpMethod(t string) bool {
```

Comment:
Why can't we have just `startsWithHTTPMethod` as it was before with the regex `^[A-Z]+`?